holonote/annotate/display.py

```diff
@@ -423,13 +421,21 @@ def get_indices_by_position(self, **inputs) -> list[Any]:
             iter_mask = (
                 (df[f"start[{k}]"] <= v) & (v < df[f"end[{k}]"]) for k, v in inputs.items()
             )
+            subset = reduce(np.logical_and, iter_mask)
+            out = list(df[subset].index)
         elif "point" in self.region_format:
-            iter_mask = ((df[f"point[{k}]"] - v).abs().argmin() for k, v in inputs.items())
+            xk, yk = list(inputs.keys())
+            distance = (
+                (df[f"point[{xk}]"] - inputs[xk]) ** 2 + (df[f"point[{yk}]"] - inputs[yk]) ** 2
+            ) ** 0.5
+            if (distance > inputs[xk] / 1e2).all():
```

Comment:
This is not ideal. This is tailored to a concrete example.

Reply:
`1e2`

Reply:
Added a `nearest_2d_point_threshold`